cdap-api/src/test/java/io/cdap/cdap/api/plugin/PluginClassTest.java

```diff
@@ -33,8 +33,9 @@
   public void testOldFormat() {
     // test that old json representation which does not have a requirements field can be be converted to Plugin class
     // by normal GSON without any custom deserializer (CDAP-14515)
-    PluginClass pluginClass = new PluginClass("sparkprogram", "wordcount", "desc", "className", "config",
-                                              Collections.emptyMap(), null);
+    PluginClass pluginClass = new PluginClass("sparkprogram", "wordcount", "className", "config",
```

Comment:
would be good to update all of these constructors to use the builder. Want to avoid calling any deprecated methods.